docs/tutorials/first-steps.rst

```diff
@@ -29,6 +29,7 @@ We'll start with a relatively simple data model:
 
 Using the `httpie <http://httpie.org>` tool we can post a sample record in the
 ``tasks`` collection:
+For proxy issues we can use  "http --proxy=http:http://usrname:passwrd@host:port COMMAND" instead of "http COMMAND"
```

Comment:
Please consider adding a note stating that:

Please [consider reading httpie documentation](https://github.com/jkbrzt/httpie#proxies) for more information If you need to configure a proxy for instance.